### PR TITLE
Add landing page aa1 to implement aa1

### PR DESCRIPTION
**Pull request by pinterid**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no landing page template for displaying:
  + The throughput chart of the last week
  + The burn down chart of the last week
  + The latest feature collection
  + The latest definition of done
  + The latest status report

**What is the new behavior (if this is a feature change)?**
- Now there is a landing page template for displaying:
  + The throughput chart of the last week
  + The burn down chart of the last week
  + The latest feature collection
  + The latest definition of done
  + The latest status report

**Other information**:
- Ref: https://getbootstrap.com/
- Ref: https://www.chartjs.org/

Here a screenshot of the implemented changes:
image

### CODE REVIEW

File: templates/footer.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+  <p></p>
```
Comment (Aichnerc) [inline templates/footer.html]: Why?

**Review by Aichnerc — APPROVED** [review]
Is ok, but check my note.

File: templates/footer.html (hunk 1 above)
Comment (kleberbaum) [inline templates/footer.html]:
```suggestion
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Review by schettn — APPROVED** [review]
lgtm

**Pull request merged**